app/models/task.rb

```diff
@@ -0,0 +1,20 @@
+# == Schema Information
+#
+# Table name: tasks
+#
+#  id          :uuid             not null, primary key
+#  name        :string
+#  description :text
+#  state       :integer
+#  project_id  :uuid
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#
+
+class Task < ActiveRecord::Base
+  belongs_to :project
+
+  enum state: %i{todo in-progress done}
+
+  validates :project, presence: true
```

Comment:
Validating presence on `project` is great! Love that you did it on `project` and not `project_id`.

It might be good to have some more validations though. A task probably should always have a state and a name likely makes sense as well.